### ISSUE TITLE
Initial difficulties

### ISSUE THREAD

**Issue by cflurin** [Steveorevo/node-red-contrib-actionflows#2]
Hi,
I've installed your node and imported the first basic flow.
It seems that it doesn't work.

actionflows v1.2.1
node-red v0.18.2
node.js v8.9.4

**Comment by cflurin** [comment]
Ok, I had to restart node-red after installing with manage palette.
Now I got a warning:
```sh
12 Feb 16:06:11 - [warn] [actionflows:action] Action cycle execution time: 0s and 4.030676ms
```

**Comment by cflurin** [comment]
Ahh concerning warning, I see how it works.

**Comment by cflurin** [comment]
Sorry, but there are some issues. Would you please test your code.
E.g. deselect `Debug action cycle execution time` and select again.

**Comment by Steveorevo** [comment]
Unfortunately not able to reproduce your issue. Could you provide a screenshot and your platform details? Operating system, version, etc.

**Comment by cflurin** [comment]
I'm running node-red on a Raspberry pi 3
```sh
Linux rpi4 4.9.59-v7+ #1047 SMP Sun Oct 29 12:19:23 GMT 2017 armv7l GNU/Linux
```
1. The basic example works as described, also I get "Hello Mars, and Solar System!" in the debug tab.
2. deselect the `Debug action cycle execution time` in the action node
3. deploy
4. click on the inject node > this time I only get "Hello World"

**Comment by Steveorevo** [comment]
Did you change any other attributes, such as the name? Still unable to reproduce; could you provide a screenshot of the node wire layout?

**Comment by cflurin** [comment]
I got it: you have to do a `Full` Deploy
Generally I use `Modified Nodes` Deploy

Maybe you add it to your documentation. By the way your README is great!

**Comment by Steveorevo** [comment]
This issue is now fixed in release 2.0.0 along with project switching. ActionFlows remap connections on partial and full deploys. A full deploy maybe required if sub or sub-subflows names have been changed when using protected scopes; a note about it is now in the docs.

**Comment by cflurin** [comment]
Hi,
Thanks for your Update.
I've installed the node on a test RPi and I'll test the functionality in the next days.
